dts/bindings/spi/ti,stellaris-spi.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2018, Zilogic Systems
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Ti Stellaris
```

Comment:
Ti -> TI

Reply:
Renamed ti,stellaris-spi.yaml to arm,pl022.yaml, so this changes not required.